contracts/V1/DataTypes.sol

```diff
@@ -111,6 +113,19 @@ struct LeaveChannel {
     bytes data; //custom data
 }
 
+
+struct ExitChannel {
+    address participant;
+    bool isPartialExit;
```

Comment:
why do we need this one? @lukastanisic99 
We already have the amount so we should already know its its partial or not

Reply:
I commented on this in some other PR also - We don't need it as you noticed, since the state machine does all the accounting internally and 'knows' is it a partial exit or a full exit (should it keep the address in the participant set or not) - Originally we were keeping track about this outside of the state machine, so it would have been useful in that case, but since we can just call 'getParticipants' on the state machine, we don't beed this information